ansible/roles/controller/tasks/deploy.yml

```diff
@@ -189,6 +189,7 @@
       "CONFIG_whisk_timeLimit_min": "{{ limit_action_time_min | default() }}"
       "CONFIG_whisk_timeLimit_max": "{{ limit_action_time_max | default() }}"
       "CONFIG_whisk_timeLimit_std": "{{ limit_action_time_std | default() }}"
+      "CONFIG_whisk_timeLimit_timeoutfactor": "{{ limit_action_timeoutfactor | default() }}"
```

Comment:
Should this rather be part of the loadbalancer config since it is specific to the loadbalancer after all?